docs/experimental-machine-learning/experimental-detections.md

```diff
@@ -0,0 +1,23 @@
+# Experimental ML Jobs and Rules
+
+Once data is being enriched, there are some rules and ML jobs which can leverage the enriched fields. 
+The experimental rules and jobs will be staged separate from the model bundle under the [releases](https://github.com/elastic/detection-rules/releases) 
+as `ML-experimental-detections-YYYMMDD-N`. 
+
+These releases will be additive, in that rule updates will come out in new releases.
+
+Note that if a rule is of `type = "machine_learning"`, then it may be dependent on a uploading and running a machine
```

Comment:
another awkward sentence. i'm going to pause on the grammar review and recommend that these docs get proofread and adjusted. I don't personally use grammarly, but I think it would help out a lot.